test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Feeds/MultiSourcePackageMetadataProviderTests.cs

```diff
@@ -69,6 +69,35 @@ public async Task GetLocalPackageMetadataAsync_WhenGlobalSourceHasPackage_Withou
                 Assert.Null(await metadata.GetDeprecationMetadataAsync());
             }
 
+            [Fact]
+            public async Task GetLocalPackageMetadataAsync_WhenGlobalSourceHasPackage_WithoutVulnerabilityMetadata()
+            {
+                // Arrange
+                var emptyTestMetadata = PackageSearchMetadataBuilder.FromIdentity(TestPackageIdentity).Build();
+
+                Mock.Get(_globalMetadataResource)
+                    .Setup(x => x.GetMetadataAsync(TestPackageIdentity.Id, true, true, It.IsAny<SourceCacheContext>(), It.IsAny<Common.ILogger>(), It.IsAny<CancellationToken>()))
+                    .ReturnsAsync(new[] { emptyTestMetadata });
+
+                Mock.Get(_metadataResource)
+                    .Setup(x => x.GetMetadataAsync(TestPackageIdentity.Id, true, false, It.IsAny<SourceCacheContext>(), It.IsAny<Common.ILogger>(), It.IsAny<CancellationToken>()))
+                    .ReturnsAsync(new[] { emptyTestMetadata });
+
+                // Act
+                var metadata = await _target.GetLocalPackageMetadataAsync(
```

Comment:
Are you sure this is testing `WhenGlobalSourceHasPackage` ? I think calling `GetLocal` just means this test is verifying local sources never contain vulnerability metadata.